test/scalability/runner/recorder/recorder.go

```diff
@@ -18,27 +18,138 @@ package recorder
 
 import (
 	"context"
+	"encoding/csv"
+	"os"
+	"strconv"
 	"sync/atomic"
 	"time"
 
 	apimeta "k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/yaml"
 
 	kueue "sigs.k8s.io/kueue/apis/kueue/v1beta1"
+	"sigs.k8s.io/kueue/test/scalability/runner/generator"
 )
 
+type event interface {
+	GetEventInfo() string
+}
+
 type CQStatus struct {
-	Name               string
+}
+
+type CQEvent struct {
+	Time      time.Time
+	Name      string
+	ClassName string
+	Cohort    string
+	UID       types.UID
+
+	CPUReservation     int64
+	CPUUsage           int64
+	CPUQuota           int64
 	PendingWorkloads   int32
 	ReservingWorkloads int32
+	AdmittedWorkloads  int32
 	Active             bool
 }
 
-type Store map[string]CQStatus
+func (e *CQEvent) GetEventInfo() string {
```

Comment:
nit: this seems unused?

Reply:
It's just to fit the `event` interface

Reply:
I see, could we have just `type event interface{}` if we don't use the function I guess.

Reply:
done